lib/theme_check/checks/missing_enable_comment.rb

```diff
@@ -26,7 +26,11 @@ def after_document(node)
         checks_missing_end_index.join(', ') + " " + (checks_missing_end_index.size == 1 ? "was" : "were")
       end
 
-      add_offense("#{message} disabled but not re-enabled with theme-check-enable", node: node)
+      add_offense("#{message} disabled but not re-enabled with theme-check-enable", node: node) do
+        script = node.source.to_enum(:scan, /(?<comment_open>{% comment %}theme-check-disable)(?<checks>.*?)(?<comment_close>{% endcomment %})(?<enclosed>[^\n]*\n[^\n]*\n)/m).map { Regexp.last_match }[-1]
+        next if script.nil?
+        node.source.insert(script.end(:enclosed), "{% comment %}theme-check-enable#{checks_missing_end_index.include?(:all) ? "" : " #{checks_missing_end_index.join(', ')}"}{% endcomment %}\n")
```

Comment:
I think you'll get the last line of the file if you append `$` at the end of your regex instead.

Reply:
I believe appending $ at the end of the regex will match the pattern at the end of the string. In this case, I'm looking to get the MatchData for the last occurrence of the pattern in the file. This might not be at the very end of the file so I don't think $ would work.

Reply:
It's the `to_enum` part that bothers me. It's creating a Fiber underneath for no reason.
So either `/Z` or `.scan(...).last`